pkg/vars/evaluator.go

```diff
@@ -11,6 +11,7 @@ var varsRegex = regexp.MustCompile(`\{\{\s*(.+?)\s*\}\}`)
 
 type Evaluator interface {
 	Eval(req *mock.Request, m *mock.Definition)
+	EvalCallback(req *mock.Request, m *mock.Definition)
```

Comment:
There is a lot of repeated code in this file. In the end, you only need to replace the tags inside the body of the callback, I don't think you need all these extra functions.

Reply:
Updated - removed extra functions, inlined into `EvalCallback`